<?xml version="1.0"?>

<launch>
  <!--控制-->
    <arg name="debug_mode"  default="false" doc="show debug info (bool)"/> 
    <arg name="pwm_radian"  default="450" doc="non scaled factor pwm/radian (steering)"/>
    <arg name="neutral_pt"  default="-5.0"   doc="steering neutral point (degree)"/>
    <arg name="use_imu"     default="true" doc="use imu for ekf (bool)"/> 
    <arg name="time_out"    default="1.0"   doc="communication time out, unit: sec"/> 
    <arg name="output"      default="log"   doc="log or screen"/> 
    <!-- for amcl -->    
    <arg name="init_x" default="0.0" />
    <arg name="init_y" default="0.0" />
    <arg name="init_a" default="0.0" />
   

    <!--  ****** Pure Pursuit ******  -->
    <node name="Pure_Pursuit" pkg="hypharos_minicar" type="Pure_Pursuit" output="screen">
        <rosparam file="$(find hypharos_minicar)/launch/params/pure_pursuit/pure_pursuit_params.yaml" command="load" />
        <remap from="/pure_pursuit/odom" to="/odom" />
        <remap from="/pure_pursuit/global_plan" to="/move_base/TrajectoryPlannerROS/global_plan" />
        <remap from="/pure_pursuit/goal" to="/move_base_simple/goal" />
        <remap from="/pure_pursuit/ackermann_cmd" to="/ackermann_cmd" />
    </node>
</launch>
